### ISSUE TITLE
CommandBar not focusable in Safari

### ISSUE THREAD

**Issue by nirtz89** [microsoft/fluentui#21114]
### Environment Information

- **Package version(s)**: latest
- **Browser and OS versions**: MacOS 12.0.1, Safari 15.1

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/Nirtz89/pen/bGooVEe

#### Actual behavior:

CommandBar is not focusable on Safari

#### Expected behavior:

CommandBar should be focusable on Safari

### Documentation describing expected behavior

Try and tab the CommandBar, it does not work.

**Comment by gouttierre** [comment]
@nirtz89 - Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows.

@spmonahan - Would you be able to confirm if this is a regression, or if this behavior is an issue? 
cc @smhigley

**Comment by smhigley** [comment]
This isn't coming from Fluent, it's caused by not enabling tab navigation to all elements in Safari's preferences, which is necessary if you want to tab to things like links and buttons in Safari on macOS: https://osxdaily.com/2011/08/14/enable-tab-key-navigation-safari/

It doesn't reproduce on my machine, where I have that setting turned on.

Closing this issue, but feel free to re-open if there are any other questions.
